eth/protocols/eth/handlers.go

```diff
@@ -273,6 +283,42 @@ func ServiceGetReceiptsQuery(chain *core.BlockChain, query GetReceiptsRequest) [
 	return receipts
 }
 
+// serviceGetReceiptsQuery69 assembles the response to a receipt query.
+// It does not send the bloom filters for the receipts
+func serviceGetReceiptsQuery69(chain *core.BlockChain, query GetReceiptsRequest) []rlp.RawValue {
```

Comment:
If you add a method with 69 in there for a new version, pls also suffix the old one with 68, otherwise it will read as universal instead of 1 version specific.